delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storages/storage/model/StorageValue.scala

```diff
@@ -0,0 +1,78 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storages.storage.model
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.AuthToken
+import ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.Permission
+
+import java.nio.file.Path
+
+sealed trait StorageValue extends Product with Serializable {
```

Comment:
other common fields could be extracted here, like algorithm etc.

Reply:
When needed. So far not needed.